app/scripts/app.js

```diff
@@ -31,10 +31,10 @@ var exports = (function() {
         router = new VueRouter();
 
         router.map({
-          "/search": {
-            name: "search",
-            component: SearchColumns
-          },
+          // "/search": {
+          //   name: "search",
+          //   component: SearchColumns
+          // },
           "/scenarios/:scenarioid/models/:modelid": {
```

Comment:
Is this url still in use? If so, should it still show the FinderColumns view or should it now also have SearchColumns as component?